tests/helpers.py

```diff
@@ -138,6 +146,34 @@ def create_ceph_block_pool():
     return cbp_obj
 
 
+def create_ceph_fs(fs_name=None):
```

Comment:
this function is already implemented - tests.helpers.create_cephfilesystem

Reply:
Removed create_ceph_fs()